code/modules/projectiles/guns/throw.dm

```diff
@@ -82,7 +82,7 @@
 	to_launch = null
 	modify_projectile(I)
 	playsound(user, fire_sound, 50, 1)
-	I.throw_at(target, get_throwrange(), get_throwspeed(), user, 1)
+	I.throw_at(target, get_throwrange(), get_throwspeed(), user, FALSE)
```

Comment:
I don't think bows are supposed to make their projectiles spin, so I changed this... but I could be wrong.